tests/phpunit/TestFramework/Coverage/JUnit/JUnitTestCaseSorterTest.php

```diff
@@ -53,9 +61,8 @@ public function test_it_returns_first_file_name_if_there_is_only_one(): void
 
         $sorter = new JUnitTestCaseSorter();
 
-        $uniqueSortedFileNames = iterator_to_array(
-            $sorter->getUniqueSortedFileNames($coverageTestCases)
-        );
+        /** @var array $uniqueSortedFileNames */
+        $uniqueSortedFileNames = $sorter->getUniqueSortedFileNames($coverageTestCases);
 
         $this->assertCount(1, $uniqueSortedFileNames);
         $this->assertSame('/path/to/test-file-1', $uniqueSortedFileNames[0]);
```

Comment:
Might be better to check the whole array here? (`assertSame()`) - that also removes the need of adding `/** @var array $uniqueSortedFileNames */` above